src/renderers/shared/ReactDebugTool.js

```diff
@@ -41,11 +41,13 @@ function emitEvent(handlerFunctionName, arg1, arg2, arg3, arg4, arg5) {
 
 var isProfiling = false;
 var flushHistory = [];
+var lifeCycleTimerStack = [];
 var currentFlushNesting = 0;
 var currentFlushMeasurements = null;
```

Comment:
It might be simpler to just store a reference to `currentTimer` here instead of leaving them split up.